Automattic/wp-calypso
# DIFM: Fetch themes from the theme showcase

#### Changes proposed in this Pull Request

Context: pdh1Xd-GP-p2#comment-953

* We want to let go of our [static theme list](https://github.com/Automattic/wp-calypso/blob/25edea7bc31526d2bcd46d73e56cefb220a3098a/client/signup/steps/difm-design-picker/themes.tsx) and fetch themes from the server. This PR modifies the theme filter for the design picker step to `do-it-for-me` for the DIFM flow.

The `DO NOT MERGE` label has been added to this PR since this needs to be merged only after the `signup/redesigned-difm-flow` feature flag is switched to `true` and all themes listed in pdh1Xd-GP-p2 are tagged with the `do-it-for-me` feature in the theme showcase.

#### Testing instructions



* Go to `/start/do-it-for-me`.
* Select "New Site".
* On the Design Picker step, confirm that themes tagged with the `do-it-for-me` feature are shown. Currently, only one theme (Balasana) has this feature.




Related to 0-as-1201933307121191/f
